docs/source/quickstart.rst

```diff
@@ -27,7 +27,7 @@ The result looks like `this <api/pydoctor.html>`_.
         --html-viewsource-base=https://github.com/twisted/pydoctor/tree/20.7.2 \
         --make-html \
         --html-output=docs/api \
-        --project-base-dir="$(pwd)" \
+        --project-base-dir="." \
```

Comment:
Does it work with relative paths ?

```suggestion
        --project-base-dir="/home/johnd/projectx/" \
```

Reply:
Yes it does work with relative paths.

Reply:
perfect. Feel free to merge it :)

In the future, I guess that `--project-base-dir` can default to current working directory so that we don't have to explicitly use it.

Reply:
See #441